src/main.js

```diff
@@ -48,26 +48,28 @@ function removeCard(event) {
     }
   };
 
-function toggleStars(event) {
-  if (event.target.src == "file:///Users/brigette.doelp/turing/mod1repeat/ideabox-boilerplate/assets/star.svg") {
-    event.target.src = "file:///Users/brigette.doelp/turing/mod1repeat/ideabox-boilerplate/assets/star-active.svg"
-  } else {
-    event.target.src = "file:///Users/brigette.doelp/turing/mod1repeat/ideabox-boilerplate/assets/star.svg"
+
+function toggleStars(event){
+  var selectedIdea = event.target.closest(".idea-box");
+  var selectedID = event.target.id;
+    for (var i = 0; i < savedIdeas.length; i++) {
+      if (savedIdeas[i].id == selectedID) {
+        event.target.src = savedIdeas[i].updateIdea();
```

Comment:
General idea for refactoring here, when we invoke `updateIdea()` why not just pass the event argument in and use the information it holds to adjust the DOM.